CODE_OF_CONDUCT.md

```diff
@@ -1,98 +1,38 @@
-  limited to members of any race, ethnicity, culture, national origin, colour,
-  immigration status, social and economic class, educational level, sex, sexual
-  orientation, gender identity and expression, age, size, family status,
-  political belief, religion, and mental and physical ability.
-
-* **Be considerate**: Your work will be used by other people, and you in turn
-  will depend on the work of others. Any decision you take will affect users
-  and colleagues, and you should take those consequences into account when
-  making decisions. Remember that we're a world-wide community, so you might
-  not be communicating in someone else's primary language.
-
-* **Be respectful**: Not all of us will agree all the time, but disagreement is
-  no excuse for poor behavior and poor manners. We might all experience some
-  frustration now and then, but we cannot allow that frustration to turn into a
-  personal attack. It’s important to remember that a community where people
-  feel uncomfortable or threatened is not a productive one.
-
-* **Be careful in the words that we choose**: We are a community of
-  professionals, and we conduct ourselves professionally. Be kind to others. Do
-  not insult or put down other participants. Harassment and other exclusionary
-  behavior aren't acceptable. This includes, but is not limited to: Violent
-  threats or language directed against another person, Discriminatory jokes and
-  language, Posting sexually explicit or violent material, Posting (or
-  threatening to post) other people’s personally identifying information
-  (“doxing”), Personal insults, especially those using racist or sexist terms,
-  Unwelcome sexual attention, Advocating for, or encouraging, any of the above
-  behavior, Repeated harassment of others. In general, if someone asks you to
-  stop, then stop.
-
-* **Try to understand why we disagree**: Disagreements, both social and
-  technical, happen all the time. It is important that we resolve disagreements
-  and differing views constructively. Remember that we’re different. Diversity
-  contributes to the strength of our community, which is composed of people
-  from a wide range of backgrounds. Different people have different
-  perspectives on issues. Being unable to understand why someone holds a
-  viewpoint doesn’t mean that they’re wrong. Don’t forget that it is human to
-  err and blaming each other doesn’t get us anywhere. Instead, focus on helping
-  to resolve issues and learning from mistakes.
+- **Be friendly and patient.**
+
+- **Be welcoming**: We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, colour, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
```

Comment:
Are there any reason to remove the line breaks here? Diffs can look prettier if more breaks are included, imho.

Reply:
I wanted to try a tool to autoformat the content but that did not work as I expected. I'll fix that